__tests__/app.test.js

```diff
@@ -204,7 +204,7 @@ describe('POST /api/reviews/:review_id/comments', () => {
     .send(invalidComment)
     .expect(404)
     .then(({ body: { msg } }) => {
-      expect(msg).toBe('Path Not Found');
+      expect(msg).toBe("Resource Not Found");
```

Comment:
Its really confusing for the reviewer to handle amendments from pervious feedback, please do this on the appropriate branch so it both keeps your git history clean, and allows us to provide effective feedback on the ticket that you have been working on